sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -21,6 +21,7 @@
 DISABLE_LOGGING_ARG = "disable_logging"
 DISABLE_METRICS_ARG = "disable_metrics"
 DISABLE_TRACING_ARG = "disable_tracing"
+DISTRO_VERSION = "distro_version"
```

Comment:
Pull from exporter constant instead so it's safe from breaking change.

Reply:
That sounds good. I can make the change in a separate PR because the exporter needs to be released first with the constant for the distro to reference. Will leave a TODO